### PR TITLE
update with Quiz 3 answers

### PR DESCRIPTION
**Pull request by jenpen**

### CODE REVIEW

**Comment by jenpen** [comment]
also realize that re: the question on how to we indicate in AR a one-to-many relationship, we would also make sure the models includ has_many /  belongs_to statements.

File: README.md [hunk 1]
```diff
@@ -49,7 +53,8 @@ add her to the list of guests in the castle.
 
 Write your code here:
 ```ruby
-# code here
+town[:residents].delete("Belle")
+town[:castle][:guests].push("Belle")
 ```
```
Comment (andrewsunglaekim) [inline README.md:58]: This is fine, and it works. But there's something fishy about it. Is the thing that you are deleting and pushing the same object?

File: README.md [hunk 2]
```diff
@@ -140,16 +192,43 @@ would look like for such an application.
...
+put "/warriors/:id" do
+    @warrior = Warrior.find(params[:id])
+    @warrior.update(params[:warrior])
+    redirect("/warriors/#{@warrior.id}")
+end
+
+PATCH '/warriors/:id' do 
+    @warrior = Warrior.create(params[:warrior])
+    redirect("/warriors")
+end
```
Comment (andrewsunglaekim) [inline README.md:224]: this looks like a creation not an update. How are put and patch different?

**Comment by andrewsunglaekim** [comment]
Great job jen :+1:

**Pull request closed without merging**